pkg/devices/coils.go

```diff
@@ -0,0 +1,86 @@
+package devices
```

Comment:
There will be a diff here between your code and my code. Please crush and commit to a branch,